app/controllers/concerns/json_renderer.rb

```diff
@@ -6,7 +6,6 @@ class ForbiddenIncludesError < RuntimeError
   class << self
     # make sure we are not including sensitive/unsupported associations
     def validate_includes(requested, allowed)
-      return if allowed == :all
```

Comment:
deploy.user is no association ... so we cannot allow `all` includes to be called ... they could be things like `delete_all`